jak-na-konflikty.md

```diff
@@ -0,0 +1,77 @@
+# Jak zvládat konflikty s kolegou?
+
+Cílem této příručky je pomoct ti zvládat konfliktní situace mezi tebou a ostatními Appliftery.
+
+V Kodexu Appliftera se můžeš dočíst, že řešení konfliktů je přirozenou součástí každé organizace. Proto je důležité, abys věděl, jak ke konfliktu přistupovat tak, abys minimalizoval jeho toxicitu pro tebe a pro tvého kolegu a zároveň aby tvé jednání směřovalo k jeho vyřešení.
+
+## Příprava
+
+Před tím, než začneš s kolegou konflikt řešit, je dobré si v hlavě či na papíře ujasnit pár věcí. 
+
+**Cílem přípravy není mít co nejvíc argumentů k tomu, abys svůj protějšek ubil argumenty.** Cílem je schopnost nahlédnout na danou věc z více úhlů, vžít se do svého kolegy a pochopit jeho motivaci a chování.
+
+- Co je věcným "odemočněným" předmětem konfliktu? Jak se daný problém dá popsat co nejvíc věcně?
+- Jak bude předmět konfliktu vnímat a vidět tvůj kolega?
+- Jaké kolem toho máš emoce? Jak se cítíš?
+- Jaké kolem toho bude mít emoce tvůj kolega? 
+- Jaké asi byly motivace kolegy? Proč postupoval tak jak postupoval?
+
+## Před konfliktem
+
+Pokud máš s nadcházejícím konfliktem spojených hodně negativních emocí, zkus je nejdříve zmírnit. Sedni si sám v tiché místnosti na 5 minut a zamedituj si o tvém kolegovi. Soustřeď se na jeho kladné vlastnosti a hezké věci, co jste spolu zažili. Uvědom si, že nacházející konflikt je pravděpodobně něco, čemu se za půl roku nad pivem zasmějete.
```

Comment:
Meditace v tiché místnosti není pro všechny. Možná by to tam chtělo zmínit, že takhle to máš ty. Já se naopak tichým přemejšlením vyraguju ještě víc.

Reply:
Co funguje tobě? :)

Reply:
@Vratislav Dostat se jakkoliv do pohodové nálady kdy zas věřím v to, že jsou lidi implicitně dobří a nedělají věci ze špatných důvodů. Často tím, že jdu chvíli pracovat a soustředím se na něco jiného, třeba.